### PR TITLE
fix: added title and aria-label to 'a' tag and button

### PR DESCRIPTION
**Pull request by VitaliiDC8**

### CODE REVIEW

File: src/blocks/Icons/Icons.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ const Icons = ({title, size = 's', items}: IconsBlockProps) => {
                         className={b('item')}
                         key={item.url}
                         href={item.url}
+                        aria-label={item.text}
```
Comment (NikitaCG) [inline src/blocks/Icons/Icons.tsx:32]: as a refactoring, can we do a restructuring for `item`?

File: src/navigation/components/SocialIcon/SocialIcon.tsx [hunk 2]
```diff
@@ -17,7 +17,14 @@ const SocialIcon: React.FC<NavigationSocialItemOwnProps> = ({icon, url, classNam
     const iconData = getMediaImage(icon);
 
     return (
-        <a href={url} target="_blank" rel="noopener noreferrer" className={b(null, className)}>
+        <a
+            href={url}
+            aria-label={iconData.alt}
```
Comment (NikitaCG) [inline src/navigation/components/SocialIcon/SocialIcon.tsx:22]: And in this place too

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/blocks/Icons/Icons.tsx (hunk 1 above)
Comment (VitaliiDC8) [reply src/blocks/Icons/Icons.tsx:32]: what do you mean? Could you describe it, please

Comment (VitaliiDC8) [reply src/blocks/Icons/Icons.tsx]: done

File: src/navigation/components/SocialIcon/SocialIcon.tsx (hunk 2 above)
Comment (VitaliiDC8) [reply src/navigation/components/SocialIcon/SocialIcon.tsx]: done

File: src/blocks/Icons/Icons.tsx [hunk 3]
```diff
@@ -24,17 +24,20 @@ const Icons = ({title, size = 's', items}: IconsBlockProps) => {
             {title && <Title className={b('header')} title={title} colSizes={{all: 12}} />}
             {items.map((item) => {
```
Comment (NikitaCG) [inline src/blocks/Icons/Icons.tsx:25]: {text, url}

File: src/blocks/Icons/Icons.tsx [hunk 4]
```diff
@@ -24,17 +24,20 @@ const Icons = ({title, size = 's', items}: IconsBlockProps) => {
...
+                return url ? (
                     <a
                         className={b('item')}
-                        key={item.url}
-                        href={item.url}
+                        key={url}
+                        href={url}
+                        aria-label={text}
+                        title={text}
                         {...getLinkProps(item.url, hostname)}
```
Comment (NikitaCG) [inline src/blocks/Icons/Icons.tsx:35]: in this line too

**Review by NikitaCG — APPROVED** [review]

File: src/navigation/components/SocialIcon/SocialIcon.tsx [hunk 5]
```diff
@@ -15,9 +15,17 @@ export interface NavigationSocialItemOwnProps extends NavigationSocialItem {
 
 const SocialIcon: React.FC<NavigationSocialItemOwnProps> = ({icon, url, className}) => {
     const iconData = getMediaImage(icon);
```
[the remark this answers is not in the document]
Comment (VitaliiDC8) [reply src/navigation/components/SocialIcon/SocialIcon.tsx:17]: we are using iconData object in the one level down

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**